crates/autopilot/src/xapi/mod.rs

```diff
@@ -0,0 +1,5 @@
+//! A top-level module foe client implementations to e**X**ternal **API**s.
```

Comment:
```suggestion
//! A top-level module for client implementations to e**X**ternal **API**s.
```
Also the name `xapi` seems unfortunate IMO. I didn't get what it's supposed to mean until I read this comment.
Additionally strictly speaking you wouldn't have to use it with an external API. To me this seems like a client for the public API. It's up to the user of the tool if the provided API is external or actually the one in the same cluster as the `autopilot`.

Reply:
I hate the name `xapi`... just didn't know what else to name this module... Let me try something else.

Reply:
I also don't have a great suggestion from the top of my head but I think the new name is an improvement. 👍